src/SmartComponents/Actions/ViewActions.js

```diff
@@ -50,14 +51,30 @@ class ViewActions extends Component {
 
     componentDidMount () {
         document.getElementById('root').classList.add('actions__view');
-        this.props.fetchRules({ page_size: this.state.itemsPerPage, category: this.props.match.params.type }); // eslint-disable-line camelcase
+        const riskMap = {
```

Comment:
I could see exporting this to the constants file possibly.... if it gets used again somewhere...

Reply:
I don't see another match for it at the moment, maybe in the future?